### PR TITLE
[HIBIKE] Delete old sensors

### PR DESCRIPTION
**Pull request by yqchen98**
This PR removes obsolete devices, specifically teamflag and encoders.

### CODE REVIEW

**Review by baby-bell — CHANGES_REQUESTED** [review]
You need to remove `TeamFlag` from the README as well, it's still there. Otherwise lgtm

File: hibike/README.md [hunk 1]
```diff
@@ -190,7 +190,6 @@ Device Type Enumeration:
 |         |                | 5            | v_batt     | float      | yes   | no     |
 |         |                | 6            | dv_cell2   | float      | yes   | no     |
 |         |                | 7            | dv_cell3   | float      | yes   | no     |
-|  0x05   | TeamFlag       | 0            | mode       | bool       | yes   | yes    |
 |         |                | 1            | blue       | bool       | yes   | yes    |
```
Comment (baby-bell) [inline hibike/README.md]: Remove the `TeamFlag` fields as well, please

**Review by baby-bell — CHANGES_REQUESTED** [review]
You are almost there, just need to remove the `TeamFlag` fields that are left over

**Review by tingtingchiang — APPROVED** [review]
LGTM

**Comment by tingtingchiang** [comment]
Issue #432

**Review by baby-bell — APPROVED** [review]

**Pull request merged**